src/apps/fee-list/FeeList.tsx

```diff
@@ -63,7 +63,7 @@ const FeeList: FC = () => {
       }
       const feesPostPaymentChange = getFeesInRelationToPaymentChangeDate(
         fbsFees,
-        true
+        false
```

Comment:
Cray detail 😅 
Although this is sufficient I think we should also channge the "hanging" hardcoded [change-date](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/103b4b10892a90180d48138093d70bf04dc5a3f2/src/apps/fee-list/utils/helper.ts#L8).
We should put it in the config area in core.
Because of the urgency of getting this out as soon as possible, I think it is ok to create a task for it and move along.

Reply:
But is it really something they should be able to configure? I mean the date is never gonna change. It's something that happened in the past. Why do you think it should be configurable?